x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.tsx

```diff
@@ -436,6 +436,15 @@ export const PARAMETERS_DEFINITION = {
       label: i18n.translate('xpack.idxMgmt.mappingsEditor.parameters.localeLabel', {
         defaultMessage: 'Locale',
       }),
+      helpText: () => (
+        <FormattedMessage
+          id="xpack.idxMgmt.mappingsEditor.parameters.localeHelpText"
+          defaultMessage="Example locale: {code}. Locales can have a maximum of 3 parts."
```

Comment:
I updated this to: 

```You can separate the language, country and variant using {hypen} or {underscore} characters. Locales can have a maximum of 3 separators. Example locale: {locale}.```

WDYT?

Reply:
Ah, that makes more sense to me! Thanks. I think "hypen" is a typo, and it might be worth changing "characters" to "separators" in the first sentence so we have a consistent noun. If there are three parts, then doesn't that imply 2 separators, not 3? And now I'm getting really nitpicky: should language, country, and variant be given an Oxford comma?

```
You can separate the language, country, and variant using {hyphen} or {underscore} separators. Locales can have a maximum of 2 separators. Example locale: {locale}.
```

Reply:
Good catches! Fixed.

Reply:
How about removing "you can":

To separate the language, country, and variant, use {hyphen} or {underscore} separators. Locales can have a maximum of 2 separators. Example locale: {locale}.

------

I'm a little confused by the example.  Is it

Example: locale: {locale}

or

Example: {local}

Reply:
Latest text:
<img width="703" alt="Screen Shot 2020-01-08 at 9 45 19 AM" src="https://user-images.githubusercontent.com/5226211/71987294-efec1400-31fb-11ea-8957-13b00df300f2.png">

Reply:
For locale, what do you think about making the description more positive?  Also, removing "locale" from the last sentence.  Need to replace ?? with correct word.

Text:

The locale to use when parsing dates. This is useful if your ??? will be viewed in different languages and you want the dates to update accordingly. Defaults to ROOT.

----

Let's shorten the first sentence in the hint text as follows. 

Updated text:

Separate the language, country, and variant using `-` or `_`. A maximum of 2 separators is allowed. 

Can the example use two separators?

Reply:
> This is useful if your ??? will be viewed in different languages and you want the dates to update accordingly. 

I'm not sure if this is accurate. I believe the `locale` parameter only accepts one value. So say you want to index data with dates in French, you would set the locale to `fr`. Also, as far as the `???`, I think this is primarily relevant for parsing months.

Reply:
> Can the example use two separators?

We can, although, I don't think this is as common.

Reply:
OK.  Let's use:

This is useful because months might not have the same name or abbreviation in all languages.

and the simpler example.